libraries/AP_BoardConfig/AP_BoardConfig_CAN.cpp

```diff
@@ -118,13 +118,15 @@ void AP_BoardConfig_CAN::init()
                     const_cast <AP_HAL::HAL&> (hal).can_mgr[drv_num - 1] = new Linux::CANManager;
                 #elif CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS
                     const_cast <AP_HAL::HAL&> (hal).can_mgr[drv_num - 1] = new ChibiOS::CANManager;
+                #else
+                    const_cast <AP_HAL::HAL&> (hal).can_mgr[drv_num - 1] = new SLCAN::CANManager(0);
```

Comment:
SLCAN?  Not SITL?

Reply:
Yes, its SLCAN, there's no CANManager implementation inside HAL_SITL. It just uses the CANManager of SLCAN.